lib/api/core/plugins/pluginsManager.js

```diff
@@ -1046,6 +1049,29 @@ class PluginsManager {
             : {}
         };
 
+      // load customs errors configuration file
+      if (!plugin.manifest.raw.errors) {
+        this.kuzzle.log.warn(`[${plugin.manifest.name}] This plugin doesn't have customs errors in the manifest file.`);
+      } else {
+        try {
+          loadPluginsError(plugin.manifest.name, plugin.manifest.raw.errors);
+          this.kuzzle.log.info(`[${plugin.manifest.name}] Custom errors successfully loaded.`);
+        } catch (e) {
+          if (e.message.match(/Error configuration file/i)
+              || e instanceof SyntaxError
+          ) {
+            errorsManager.throw(
+              'plugins',
+              'runtime',
+              'errors_configuration_file',
+              'plugin.manifest.name',
```

Comment:
I was wrong on my suggestion, my bad. Since there is only the value of `plugin.manifest.name`, you didn't need string interpolation
```suggestion
              plugin.manifest.name,
```